appveyor.yml

```diff
@@ -59,6 +59,12 @@ init:
 clone_depth: 1
 
 install:
+ # Ensure we have the latest version of pip
+ - py -m pip install --upgrade pip

```

Comment:
Upgrading `pip` here does not prevent warnings about it being outdated during packages installation because when creating an venv a predefined version of `pip` is used rather than one currently installed. It makes sense to update it after our venv is activated.